packages/web-react/src/components/Footer/stories/Footer.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import DocsBox from '../../../../docs/DocsBox';
+import ReadMe from '../README.md';
+import { Footer } from '..';
+
+const meta: Meta<typeof Footer> = {
+  title: 'Components/Footer',
+  component: Footer,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    elementType: {
+      control: 'text',
+      table: {
+        defaultValue: { summary: 'footer' },
+      },
+      children: {
+        control: 'object',
+      },
+    },
+  },
+  args: {
+    elementType: 'div',
+    children: <DocsBox>Footer</DocsBox>,
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Footer>;
+
+export const Playground: Story = {
+  name: 'Footer',
+};
```

Comment:
Do we want to keep it like this, add content as in the demo, or create a new composition in the storybook?